tokio-macros/src/entry.rs

```diff
@@ -354,16 +353,17 @@ fn parse_knobs(mut input: syn::ItemFn, is_test: bool, config: FinalConfig) -> To
         (start, end)
     };
 
-    let crate_name = config.crate_name.as_deref().unwrap_or("tokio");
-
-    let crate_ident = Ident::new(crate_name, last_stmt_start_span);
+    let crate_path = config
+        .crate_name
+        .map(ToTokens::into_token_stream)
+        .unwrap_or_else(|| Ident::new("tokio", last_stmt_start_span).into_token_stream());
```

Comment:
It took me some time to figure out that if you just use `quote` here without a proper span you will trigger all sorts of lints. Not using a span was bad to being with anyway. Happy to build a custom `Path` here instead if desired. But I could also just use `quote_spanned`.